.eslintrc.js

```diff
@@ -12,9 +12,18 @@ module.exports = {
   rules: {
     'import/newline-after-import': 1,
     'import/order': ["error", { "newlines-between": "always", "groups": ["builtin", ["external", "internal"], "parent", "sibling", "index"] }],
-    'prettier/prettier': ['error'],
-    'sort-imports': 'off', // No auto-fix!
+    'prettier/prettier': [
+      'error',
+      {
+        'printWidth': 150,
+        'singleQuote': true,
+        'endOfLine': 'auto',
+        'trailingComma': 'none'
```

Comment:
To be honest, I kinda like trailing commas. It makes it easier to reformat the code (change the order, add a property, etc). I don't mind the readability.

I think that is also why prettier decided to change the default value to always enforce trailing commas: https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker

@simondel what is your feeling about this?

Reply:
They look pretty ugly, but i mean, its your choice :D